### ISSUE TITLE
Add TSLint codelyzer extension

### ISSUE THREAD

**Issue by eduardomourar** [codacy/codacy-tslint#2]
* **I'm submitting a ...**
[X] feature request

* **Summary**

I am trying to add the codelyzer extension in tsling.json file:
```
"extends": ["codelyzer"]
```

But my Codacy project is ignoring the lint configuration file.

It would nice to have [codelyzer](https://github.com/mgechev/codelyzer) and [tslint-angular](https://github.com/mgechev/tslint-angular) dependencies preinstalled.

**Comment by rtfpessoa** [comment]
@eduardomourar we currently support codelizer https://github.com/codacy/codacy-tslint/blob/master/package.json#L71 but not tslint-angular. Any PRs are welcome.

**Comment by eduardomourar** [comment]
@rtfpessoa , thanks. I think the codacy services yesterday were slow, so only after a while that the linting kicked off.
